server/spec/post_bind_bonus_pool_spec.rb

```diff
@@ -409,4 +467,127 @@ def ent_derived_test(pool, expected_quantity)
     @cp.get_entitlement(ent.id)['quantity'].should == 10
   end
 
+  it 'should allow limited consumption of Bonus pools for unlimited master pool created with non-zero virt_limit' do
```

Comment:
The premise of this test is wrong. If the master pool is unlimited, the bonus pool should always be unlimited too

Reply:
Fixed.